### ISSUE TITLE
[Help Request] New issues are not being created on in project

### ISSUE THREAD

**Issue by rainrivas** [JasonEtco/todo#132]
I've attempted to configure one of my repos for use with the bot, but it doesn't seem to be creating issues.
I have the following in my .github/config.yml
```yml
todo:
keyword: "todo"
```
And in my js file
```
/**
 * @todo Take over the world
 * @body Humans are weak; Robots are strong. We must cleans the world of the virus that is humanity.
 */
function ruleOverPunyHumans () {
  // We must strategize beep boop
}
```

When I run `node bin/todo.js -o rainrivas -r rain-alm-cli -s 1f38f905a943221bf3e7ca2f4ad5d22ced42f6a5` as suggested in the README it returns blank despite seeing the "todo" comment in that [commit](https://github.com/rainrivas/rain-alm-cli/commit/1f38f905a943221bf3e7ca2f4ad5d22ced42f6a5)

I really like the idea of using this tool and any help to set this up is much appreciated.

**Comment by JasonEtco** [comment]
Hey @rainrivas, sorry its not working for you 😭 Thanks for trying the CLI to verify, that's super helpful.

Two things:

1. Make sure that `keyword: "todo"` is indented. Or, since thats the default already, you can go ahead and delete the file entirely.
2. It looks like that todo was added to a pull request. Normally, the app should create a comment on pull requests, not a new issue (until that pull request is merged to master). Why it didn't create a comment is a very good question.

Probably a dumb question, but are you sure the app is installed on your repository?

**Comment by rainrivas** [comment]
@JasonEtco yep, double checked that it is installed and configured for all repos on my profile. I'll try to remove the config file and then push a new `@todo` tomorrow to see if that works out.

**Comment by rainrivas** [comment]
@JasonEtco still no luck with this PR: https://github.com/rainrivas/rain-alm-cli/pull/2
Looks like it's not making a comment for the new file
Also I already deleted the file as suggested
>1. Make sure that keyword: "todo" is indented. Or, since thats the default already, you can go ahead and delete the file entirely.

Here is the output from the CLI which looks like it *should* generate an issue/comment
```
rain-mbp-wifi:todo efrain$ node bin/todo.js -o rainrivas -r rain-alm-cli -s cd719ec6024679d2be9cb60f7a5b950b8f71e352
---
Title: Take over the world for real this time
Body:
 Humans are weak; Robots are strong. We must cleans the world of the virus that is humanity.

---

https://github.com/rainrivas/rain-alm-cli/blob/cd719ec6024679d2be9cb60f7a5b950b8f71e352/todotest.js#L2-L7

---

###### This issue was generated by [todo](https://todo.jasonet.co) based on a `@todo` comment in cd719ec6024679d2be9cb60f7a5b950b8f71e352. It&#x27;s been assigned to @rainrivas because they committed the code.
---
```

**Comment by thulioph** [comment]
I have the same problem but without any CLI output.

**Comment by rainrivas** [comment]
@JasonEtco any clues as to what's going on with this one?

**Comment by aes421** [comment]
@rainrivas @thulioph Not sure if it will help you guys but I have traced it down to the following regex...
`^diff --git a\/.+ b\/(.+)[\\s\\S]+?^@{2}.+\\+(\\d+).+@{2}$([\\s\\S]+?(^(\\+).*(@todo|TODO)(?:[:-\\s]+)?\\s(.*))[\\s\\S]+)`

which is set in handler_setup.js.

It then goes to push-handler.js and runs this line to execute the regex and for some reason it does not match for me.  
`while ((match = regex.exec(chunk)) !== null) {`


Trying to wrap my head around the regex to figure out why.  Will update if I learn anything more.
